src/v2/Apps/NewForYou/Components/NewForYouArtworksGrid.tsx

```diff
@@ -1,80 +1,48 @@
-import { Box, Button, Spacer, Text } from "@artsy/palette"
-import React, { FC, Fragment } from "react"
-import {
-  createPaginationContainer,
-  graphql,
-  RelayPaginationProp,
-} from "react-relay"
+import { Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import React, { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
 import ArtworkGridItemFragmentContainer from "v2/Components/Artwork/GridItem"
-import { Masonry } from "v2/Components/Masonry"
 import { extractNodes } from "v2/Utils/extractNodes"
 import { NewForYouArtworksGrid_viewer } from "v2/__generated__/NewForYouArtworksGrid_viewer.graphql"
 
 interface NewForYouArtworksGridProps {
   viewer: NewForYouArtworksGrid_viewer
-  relay: RelayPaginationProp
 }
 
 export const NewForYouArtworksGrid: FC<NewForYouArtworksGridProps> = ({
   viewer,
-  relay,
 }) => {
-  const [loading, setLoading] = React.useState(false)
   const artworks = extractNodes(viewer.artworksForUser)
 
-  const loadMore = () => {
-    if (!relay.hasMore() || relay.isLoading()) return
-
-    setLoading(true)
-
-    relay.loadMore(25, err => {
-      if (err) console.error(err)
-
-      setTimeout(() => {
-        setLoading(false)
-      }, 2500)
-    })
-  }
-
   return (
     <>
       {artworks?.length > 0 ? (
-        <Masonry columnCount={[2, 3, 4]}>
+        <GridColumns gridRowGap={[2, 4]} alignItems="flex-end">
```

Comment:
We generally only use `GridColumns` for page layout. What you're looking for is [`ArtworkGrid`](https://github.com/artsy/force/tree/main/src/v2/Components/ArtworkGrid)

Reply:
fixed